### PR TITLE
Weather

### PR DESCRIPTION
**Pull request by drakaz**
This patch allow to use previmeteo instead of  google for weather plugin. Previmeteo need a (free for home use) token to work. The token is configurable in web UI.

### CODE REVIEW

**Comment by Alkorin** [comment]
Same as #39 for server/plugins/weather/plugin_weather.cpp

**Comment by drakaz** [comment]
Missing CRLF removed on linux, fixed.

File: server/plugins/weather/plugin_weather.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 Q_EXPORT_PLUGIN2(plugin_weather, PluginWeather)
 
-PluginWeather::PluginWeather():PluginInterface("weather", "Weather", BunnyZtampPlugin)
+PluginWeather::PluginWeather():PluginInterface("weather", "Meteo",BunnyZtampPlugin)
 {
```
Comment (Alkorin) [inline server/plugins/weather/plugin_weather.cpp]: Hum ?

**Comment by drakaz** [comment]
Restored english word.

**Pull request merged**